doc/storage_data_plane_commands_auth.md

```diff
@@ -0,0 +1,74 @@
+# Storage Data Plane Commands Auth
+
+Azure operations can be divided into two categories - control plane and data plane. By running storage data plane commands, requests will be sent directly to Storage service instead of being sent to ARM(Azure Resource Manager). Hence additional credentials are required for successfully auth to Storage service.
+
+## Authorization Options
+
+There are several ways to authorize for storage data plane commands:
```

Comment:
```suggestion
There are several ways to authenticate to storage data plane commands:
```

See https://english.stackexchange.com/questions/225756/is-it-authenticate-with-or-authenticate-against